source/core/src/main/com/csse3200/game/components/quests/AbstractQuest.java

```diff
@@ -2,6 +2,8 @@
 
 import com.csse3200.game.entities.Entity;
 
+import java.util.Arrays;
```

Comment:
You have a bunch of warnings for unused import statements. There are a few warnings in this file if u can fix those

Reply:
thanks. fixing now :)